www/templates/html/Peoplefinder/Developers.tpl.php

```diff
@@ -1,174 +1,117 @@
-
-<script type="text/javascript" src="scripts/jquery.beautyOfCode.js"></script>
+<?php
+$page->addHeadLink('https://cdn.jsdelivr.net/highlight.js/9.2.0/styles/solarized-dark.min.css', 'stylesheet');
+?>
 
 <script type="text/javascript">
-    $.beautyOfCode.init({
-        theme: "RDark",
-        brushes: ['Xml', 'JScript', 'CSharp', 'Plain', 'Php', "Java", "JavaFX"],
-        ready: function() {
-            $.beautyOfCode.beautifyAll();
-
-        }
-    });
+    require(['jquery', 'https://cdn.jsdelivr.net/highlight.js/9.2.0/highlight.min.js'], function($, hljs) {
+        $('.resource pre.code code').each(function() {
+            hljs.highlightBlock(this);
+        })
+    })
 </script>
 
-<div class="three_col left">
-
-    <?php
+<div class="wdn-grid-set">
+    <div class="bp3-wdn-col-three-fourths">
+        <?php
         $resource = "UNL_Peoplefinder_Developers_" . $context->resource;
         $resource = new $resource;
         ?>
         <div class="resource">
-        <h1 id="instance" class="sec_main"><?php echo $resource->title; ?> Resource</h1>
-        <h3>Details</h3>
-        <ul>
-            <li>
-                <h4 id="instance-uri"><a href="#instance-uri">Resource URI</a></h4>
-                <blockquote>
-                    <?php
-                    $uri = $resource->uri;
-                    if (substr($uri, 0, 2) == '//') {
-                        $uri = 'http:' . $uri;
-                    }
-                    ?>
-                    <p><?php echo $uri; ?></p>
-                </blockquote>
-            </li>
-            <li>
-                <h4 id="instance-properties"><a href="#instance-properties">Resource Properties</a></h4>
-                <table class="zentable neutral">
-                <thead><tr><th>Property</th><th>Description</th><th>JSON</th><th>XML</th></tr></thead>
-                  <tbody>
-                  <?php
-                    foreach ($resource->properties as $property) {
-                      echo "<tr>
-                                <td>$property[0]</td>
-                                <td>$property[1]</td>
-                                <td>$property[2]</td>
-                                <td>$property[3]</td>
-                            </tr>";
-                    }
-                  ?>
-                  </tbody>
-                </table>
-            </li>
-            <li>
-                <h4 id="instance-get"><a href="#instance-get">HTTP GET</a></h4>
-                <p>Returns a representation of the resource, including the properties above.</p>
-            </li>
-            <li>
-                <h4 id="instance-get-example-1"><a href="#instance-get-example-1">Example</a></h4>
-                <ul class="wdn_tabs">
+            <h1 id="instance" class="sec_main">API: <?php echo $resource->title; ?> Resource</h1>
+
+            <h2 id="instance-uri"><a href="#instance-uri">Resource URI</a></h2>
+            <blockquote>
```

Comment:
good catch. I was so overwhelmed by everything else that was bad that I missed this one. WHO WROTE THIS!? (me)